src/frontend/components/app/language-select/language-select.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box, Button } from '@adminjs/design-system'
+import React, { FC, useCallback } from 'react'
+import { locales } from '../../../../locale'
+
+const LanguageSelect: FC = () => {
+  const handleButton = useCallback((lang) => {
+    window.localStorage.setItem('locale', JSON.stringify(locales[lang]))
+    window.location.reload(true)
+  }, [])
+
+  return (
+    <Box flex>
+      <Button onClick={(): void => handleButton('pl')}>PL</Button>
```

Comment:
I think this shouldn't be hardcoded, available languages should be chosen from some kind of configuration, preferably `locale.availableLanguages` in `AdminJSOptions`. You can then save them in redux store in `initializeStore` function.  The store setup function would have an additional requirement which checks whether languages are actually existing.
The reason for making this configurable is that if people are creating custom translations and they may not want to translate everything to every possible language.

Explicit return type `: void` should also be unnecessary since React can derive that